platform/jewel/ide-laf-bridge/src/main/kotlin/org/jetbrains/jewel/bridge/SwingBridgeService.kt

```diff
@@ -15,15 +22,31 @@ import org.jetbrains.jewel.ui.ComponentStyling
 import org.jetbrains.jewel.ui.component.copyWithSize
 
 @Suppress("UnstableApiUsage")
-internal class SwingBridgeService {
+internal class SwingBridgeService : Disposable {
+    // TODO Pick the proper scope
+    private val coroutineScope = CoroutineScope(SupervisorJob() + Dispatchers.Default)
```

Comment:
@rock3r I need your input on this with all your knowledge about running coroutines in IJP 🙏

Reply:
As per the [IJP docs](https://plugins.jetbrains.com/docs/intellij/coroutine-scopes.html#using-a-correct-scope), this is the easiest way:

```kotlin
internal class SwingBridgeService(scope: CoroutineScope) {
```

No need for `Disposable`, either; the platform should take care of it.